databuilder/extractor/bigquery_usage_extractor.py

```diff
@@ -38,12 +41,19 @@ class BigQueryTableUsageExtractor(Extractor):
     def init(self, conf):
         # type: (ConfigTree) -> None
         self.key_path = conf.get_string(BigQueryTableUsageExtractor.KEY_PATH_KEY, None)
+        self.cred_key = conf.get_string(BigQueryTableUsageExtractor.CRED_KEY, None)
         if self.key_path:
             credentials = (
                 google.oauth2.service_account.Credentials.from_service_account_file(
                     self.key_path, scopes=BigQueryTableUsageExtractor._DEFAULT_SCOPES))
         else:
-            credentials, _ = google.auth.default(scopes=BigQueryTableUsageExtractor._DEFAULT_SCOPES)
+            if self.cred_key:
```

Comment:
Minor: should this be a `else if` at top level for more readability and move the below else also one level .

Reply:
sure, updated